docs/deployment-manager-guide.md

```diff
@@ -1,65 +1,21 @@
-specification. See an example [here](../examples/deployment-manager/autogen/singlevm/README.md).
-1. Use `mpdev cfg set` to customize values in the retrieved autogen
-specification, or edit the spec manually.
-3. Execute `mpdev apply` to generate the Deployment Manager template.
-
-## Upload solution to Partner Portal
-
-Open [Partner Portal](https://console.cloud.google.com/partner/solutions) and 
-select your solution from the list of solutions, then follow these steps:
-
-1. Next to **Deployment Package**, click **Edit**.
-1. Select **Upload a Package**, and then click **Continue**
-1. Select the deployment package created by `mpdev` for **Upload a package**,
-uncheck the **Metadata selection** box and then click **Continue**.
-1. Click **Save** to save changes.
+To learn how to create a deployment package and submit the package to Google, see the Google Cloud Marketplace documentation for [Creating your deployment package](https://cloud.google.com/marketplace/docs/partners/vm/create-deployment-package).
 
-**WARNING:** Unchecking the **Metadata selection** box is crucial, so that
-`mpdev` does not override the solution metadata configured in earlier steps of
-the Partner Portal wizard.
+## Overview
 
-## Upload product to Producer Portal
-**Preview**: This product or feature is covered by the [Pre-GA Offerings Terms](https://cloud.devsite.corp.google.com/terms/service-terms#1) of the Google Cloud Platform Terms of Service. Pre-GA products and features may have limited support, and changes to pre-GA products and features may not be compatible with other pre-GA versions. For more information, see the [launch stage descriptions](https://cloud.devsite.corp.google.com/products#product-launch-stages).
+You use the following `mpdev` commands to generate a deployment package:
 
-The following commands create and configure a Google Cloud Storage bucket that you can use to upload
-your Deployment Manager template to Producer Portal.
-
-```
-BUCKET_NAME=<BUCKET_NAME>
-gsutil mb $BUCKET_NAME
-gsutil versioning set on gs://$BUCKET_NAME
-gsutil iam ch "group:marketplace-ops@cloud.google.com:objectViewer" $BUCKET_NAME
-```
-
-Prior to running `mpdev apply`, ensure that the `zipFilePath` that you specified
-in your [`DeploymentManagerTemplate`](../examples/deployment-manager/autogen/singlevm/configurations.yaml)
-resource points to the Google Cloud Storage bucket that you created.
-
-If following an [example](../examples/deployment-manager/autogen/singlevm) in
-this repo, you can use the following command to set the `zipFilePath` programmatically:
-```
-mpdev cfg set <DIR> zipPath gs://$BUCKET_NAME/dm-template.zip
-```
+1. Use the `mpdev pkg get` command to retrieve a [preconfigured Autogen
+specification](../examples/deployment-manager/autogen).
+1. Use `mpdev cfg set` to customize values in the retrieved Autogen
+specification, or edit the spec manually.
+3. Execute `mpdev apply` to generate the Deployment Manager template.
```

Comment:
> Execute `mpdev apply`

It made me stop and have to reread when I saw that this one was "Execute" vs the other two being "Use" - not sure if that's intentional given that it's the final application step, but it did slow me down